tests/smoke/Commands/version-help-doctor.smoke.yaml

```diff
@@ -9,6 +9,14 @@ tests:
         regex: |-
           ^Juvix version 0.3.0-([a-f0-9]{7}).*
 
+  - name: cli-numeric-version
+    command:
+      - juvix
+      - --numeric-version
+    exit-status: 0
+    stdout: |
+      0.3.0
```

Comment:
have you considred a regular expression so we don't need to update the test every version?

Reply:
We can do that, sure ;)